edit-post/assets/stylesheets/main.scss

```diff
@@ -168,6 +168,32 @@ body.gutenberg-editor-page {
 			outline-offset: 0;
 		}
 	}
+
+	input[type=checkbox] {
+		border: 2px solid $dark-gray-300;
+		border-radius: $radius-round-rectangle / 2;
+		margin-right: 12px;
+		transition: none;
+
+		&:focus {
+			border-color: $dark-gray-300;
+			box-shadow: 0 0 0 1px $dark-gray-300;
+		}
+
+		&:checked:focus {
```

Comment:
This would be more readable if it were underneath the plain `:checked` style below. I think it's nice to "stack" styles like that.

Reply:
Pushing a fix in a second, let me know if that actually addresses your comment, wasn't sure I understood you perfectly.